support-frontend/assets/components/list/list.jsx

```diff
@@ -2,16 +2,35 @@
 
 import React from 'react';
 import { css } from '@emotion/core';
-import { brandAlt } from '@guardian/src-foundations/palette';
+import { brand, brandAlt } from '@guardian/src-foundations/palette';
 import { space } from '@guardian/src-foundations';
 import { from } from '@guardian/src-foundations/mq';
 import { body } from '@guardian/src-foundations/typography';
 
-type ListPropTypes = {
-  items: Array<Object>,
+export type ListItemText = {
+  explainer: string,
```

Comment:
for a generic list, explainer seems like a strange name. how about `content` ?

Reply:
That was mainly not to cause regressions with the GW landing page, but you're right it's not a great name, it's worth changing.